index.html

```diff
@@ -0,0 +1,142 @@
+            <button class="btn">See Project</button>
+          </div>
+        </article> -->
+        <!--end of single article-->
+
+        <!--single article-->
+        <!-- <article class="flexbox order-2">
+          <div class="article-text">
+            <h2 class="project-name">Muti-post Stories</h2>
+            <p class="job">FACEBOOK <span>.Full stack Dev</span><span>.2015</span></p> -->
+            <!-- <p class="articel-context">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p> -->
+            
+            <!-- <div class="btns">
+            <button class="mini">html</button>
+            <button class="mini">Ruby on rails</button>
+            <button class="mini">css</button>
+            <button class="mini">javascript</button>
+            </div>
+            <button class="btn">See Project</button>
+          </div>
+           <img src="./images/Snapshoot Portfolio1.svg" alt="project" class="portfoilio-img">
+        </article> -->
+        <!--end of single article-->
+        <!--single article-->
+        <!-- <article class="flexbox order-3">
+          <img src="./images/Snapshoot human.svg" alt="project" class="portfoilio-img">
+          <div class="article-text">
+            <h2 class="project-name">Facebook 360</h2>
+            <p class="job">Facebook <span>.Full Stack Dev</span><span>.2015</span></p>
+            <p class="articel-context">Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
+            <div class="btns">
+            <button class="mini">html</button>
+            <button class="mini">Ruby on rails</button>
+            <button class="mini">css</button>
+            <button class="mini">javascript</button>
+            </div>
+            <button class="btn">See Project</button>
+          </div>
+        </article> -->
+        <!--end of single article-->
+        <!--single article-->
+        <!-- <article class="flexbox order-4">
+          <div class="article-text">
+            <h2 class="project-name">Uber Navigation</h2>
+            <p class="job">Uber <span>.Lead developer</span><span>.2018</span></p>
+            <p class="articel-context">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
+            <div class="btns">
+            <button class="mini">html</button>
+            <button class="mini">Ruby on rails</button>
+            <button class="mini">css</button>
+            <button class="mini">javascript</button>
+            </div>
+            <button class="btn">See Project</button>
+          </div>
+          <img src="./images/Snapshoot Portfolio2.svg" alt="project" class="portfoilio-img">
+        </article> -->
+        <!--end of single article-->
+      <!-- </div>
+      
+    </section> -->
+  </div>
```

Comment:
- [ ] I would suggest you please remove the commented code as it make the code unclean